src/helpers/daily-data-types/health-connect.tsx

```diff
@@ -76,20 +62,71 @@ const healthConnectTypeDefinitions: DailyDataTypeDefinition[] = [
     {
         type: DailyDataType.HealthConnectLightSleepMinutes,
         dataProvider: healthConnectLightSleepMinutesDataProvider,
-        availabilityCheck: simpleAvailabilityCheck(
-            "HealthConnect",
-            ["sleep"],
-            true
-        ),
-        labelKey: "light-sleep-time",
+        availabilityCheck: simpleAvailabilityCheck('HealthConnect', 'sleep', true),
+        labelKey: 'light-sleep-time',
         icon: <FontAwesomeSvgIcon icon={faBed} />,
         formatter: minutesFormatter,
         yAxisConverter: sleepYAxisConverter,
         previewDataRange: [180, 240]
+    },
+    {
+        type: DailyDataType.HealthConnectSteps,
+        dataProvider: healthConnectStepsDataProvider,
+        availabilityCheck: simpleAvailabilityCheck('HealthConnect', 'steps-daily', true),
+        labelKey: 'steps',
+        icon: <FontAwesomeSvgIcon icon={faPersonRunning} />,
```

Comment:
Can we use faFootprints and also update all the other step icons to use that one, to better differentiate when we're showing a lot of activity things.